### ISSUE TITLE
Submission Form / Tools List - list of "accessibility checks"

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-evaluation-tools-list#101]
I wonder where this list comes from and how these terms are defined? Some of these seem very specific (e.g. color contrast) while others seem extremely broad (e.g. compatible with assistive technology). I'm also wondering about "WAI-ARIA attributes", which seems to either fall under other checks (e.g. page structure, text alternatives, and compatible with assistive technologies) or not really an accessibility check in itself (wrong ARIA attributes or not directly an accessibility failure even though they will often correlate with an accessibility failure). I'm happy to retract my concern if this has been sufficiently discussed in EOWG, it just seems quite arbitrary to me.

**Comment by shawna-slh** [comment]
Vera and Kris Anne, Please add links to minutes of EOWG discussions on this topic, and any other input. Thanks!

**Comment by shawna-slh** [comment]
The list does not make sense to me.

/me edits to [HIGH]

**Comment by kakinney** [comment]
After doing my review, I agree with this issue.  I can't find any discussion of why we had this list or how it was developed, so I don't know that it's needed.

**Comment by kakinney** [comment]
[Survey where the Accessibility Checks is discussed](https://www.w3.org/2002/09/wbs/35532/EOweek3-25-22/results) - only a few people had comments about the list.

**Comment by iadawn** [comment]
From @maryjom in survey:

> I'm not sure how useful the Accessibility Checks list of filters is. There are 51 total Level A and AA requirements in WCAG. This is cherry picking and/or categorizing them (I'm not sure which). For example, "Readability" encompasses 6 WCAG criteria (1 Level A, 1 Level AA, and 4 Level AAA). "Language" is also listed which double-counts the readability list because that covers the same Level A and AA criteria. Many of the tools are checking a lot of things (IBM's included) yet it's apparent they aren't specifying in this list which checks they include. I suspect because the language used in this list doesn't match the WCAG short names for the requirements. I don't see any documentation that shows the mapping. If the goal is to differentiate robust tools from more simple tools that check only for one or two aspects of accessibility it might be better to have "Limited accessibility test" and "Robust accessibility test" choices and describe what that means in the form (tests a few WCAG SC, tests a wide range of WCAG SC). Then the tools describe what that entails in their product details.

**Comment by nitedog** [comment]
After hear the explanations during the [EOWG meeting of 30 June 2023](https://www.w3.org/2023/06/30-eo-minutes#item01) and more careful review of the list of "Accessibility checks", I suggest the following:

- [high] reword "Compatible with assistive technology" -- I don't know what this phrase means and think it probably covers the majority of WCAG checks. That is, it doesn't seem useful to users of the list.
- [medium] change "Media alternatives" to "Video and audio" -- I think the term "media" is too cryptic for many readers, and is also inaccurate (also text alternatives are media alternatives, unless you mean time-based media).
- [medium] change "Error (prevention)" to "Error messages", "Errors and feedback", "Feedback from forms", or such -- I think "error prevention" might be too jargony as well.
- [low] I'm not sure what to expect from "Navigation and links" but don't have a better suggestion at the moment.
